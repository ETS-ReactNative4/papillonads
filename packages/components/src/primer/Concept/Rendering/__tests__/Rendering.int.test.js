import {
  renderSpecificationsStandardsGuidelines,
  renderOpenSourceDependencyProjects,
  renderDesignSystemCore,
  renderProductMicrosites,
} from './Rendering.int.render'

describe('<Rendering />', () => {
  describe('Render', () => {
    test('must match renderSpecificationsStandardsGuidelines()', () => {
      expect(global.renderToJSON(renderSpecificationsStandardsGuidelines())).toMatchSnapshot()
    })
    test('must match renderOpenSourceDependencyProjects()', () => {
      expect(global.renderToJSON(renderOpenSourceDependencyProjects())).toMatchSnapshot()
    })
    test('must match renderDesignSystemCore()', () => {
      expect(global.renderToJSON(renderDesignSystemCore())).toMatchSnapshot()
    })
    test('must match renderProductMicrosites()', () => {
      expect(global.renderToJSON(renderProductMicrosites())).toMatchSnapshot()
    })
  })
})
