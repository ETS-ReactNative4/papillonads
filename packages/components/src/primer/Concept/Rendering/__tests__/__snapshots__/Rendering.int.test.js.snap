// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Rendering /> Render must match designSystemCore() 1`] = `
<div
  className={null}
>
  <h2>
    📗 Papillon Ads Implementation
  </h2>
  <h3>
    📖. Design System Core
  </h3>
  <h4>
    📃. Setup file(s)
  </h4>
  <div>
    ﹘ Coming soon.
  </div>
  <h4>
    📃. Knowledge base
  </h4>
  <div>
    ﹘ Coming soon.
  </div>
</div>
`;

exports[`<Rendering /> Render must match openSourceDependencyProjects() 1`] = `
<div
  className={null}
>
  <h2>
    📗 Open Source Dependency Projects
  </h2>
  <h3>
    📖. React
  </h3>
  📝 Coming soon 🐬
</div>
`;

exports[`<Rendering /> Render must match productMicrosites() 1`] = `
<div
  className={null}
>
  <h2>
    📗 Papillon Ads Implementation
  </h2>
  <h3>
    📖. Product Microsites
  </h3>
  <h4>
    📃. Setup file(s)
  </h4>
  <div>
    ﹘ Coming soon.
  </div>
  <h4>
    📃. Knowledge base
  </h4>
  <div>
    ﹘ Coming soon.
  </div>
</div>
`;

exports[`<Rendering /> Render must match specificationsStandardsGuidelines() 1`] = `
<div
  className={null}
>
  <h2>
    📗 Specifications/Standards/Guidelines
  </h2>
  Coming soon...
</div>
`;
